### ISSUE TITLE
Use configuration file for logging

### ISSUE THREAD

**Issue by Toilal** [Tigge/Garmin-Forerunner-610-Extractor#116]
logging could be configured using a configuration file. It would then be easier for users to enable, disable and configure the logging.

**Comment by Toilal** [comment]
Device name could be added using this answer : http://stackoverflow.com/questions/16203908/how-to-input-variables-in-logger-formatter

**Comment by Tigge** [comment]
Product name could be usable, but since only once device is active each time the program is run and that information is already present in the log from <https://github.com/Tigge/Garmin-Forerunner-610-Extractor/blob/split/antfs-cli.py#L169> I'm not sure including it on every line is necessary.

As long as the log is only a toggle to generate a debug log i think the `--debug` switch is all that is needed. It could be a bit nice to have configuration options for developers though.

**Comment by Toilal** [comment]
I think it's not a so good idea, now that log files are written in `~/.config/antfs-cli`. 

Feel free to close ...

**Comment by Tigge** [comment]
Could still be implemented in part still, but closing for now.
